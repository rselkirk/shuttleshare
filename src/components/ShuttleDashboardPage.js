import React from 'react';
import ShuttleList from './ShuttleList';

const ShuttleDashboardPage = () => (
  <div>
    <ShuttleList />
  </div>
);

export default ShuttleDashboardPage;