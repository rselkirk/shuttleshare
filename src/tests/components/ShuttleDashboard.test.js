import React from 'react';
import { shallow } from 'enzyme';
import ShuttleDashboardPage from '../../components/ShuttleDashboardPage';

test('should render ShuttleDashboardPage correctly', () => {
  const wrapper = shallow(<ShuttleDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
