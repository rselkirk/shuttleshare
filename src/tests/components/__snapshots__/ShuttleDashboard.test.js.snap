// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ShuttleDashboardPage correctly 1`] = `
<div>
  <Connect(ShuttleList) />
</div>
`;
